vector/vector.yaml

```diff
@@ -4,6 +4,15 @@ sources:
     type: docker_logs
 
 transforms:
+  docker_with_grafana_nginx_separated:
+    type: remap
+    inputs:
+      - docker
+    source: |
+      if .label."com.docker.compose.service" == "nginx" && match(string!(.message), r'^\[grafana\]') {
+        .label."com.docker.compose.service" = "nginx-grafana"
```

Comment:
I feel a little guilty about overwriting the service with a name that's not really one of our service names, but I think it's worth it to keep the main app vs Grafana NGINX logs separate.